### PR TITLE
feat: add r/gnoland/pages

### PR DESCRIPTION
**Pull request by moul**
This PR adds a new `r/gnoland/pages` realm, powered by the `p/demo/blog` engine.
It's intended to be used the same way, but only contain pages which will be updated more frequently.

The first pages will be:
- r/gnoland/pages:p/gor
- r/gnoland/pages:p/events
- r/gnoland/pages:p/tokenomics
- r/gnoland/pages:p/start

And will allow making gno.land even simpler, and just linking to these dynamic onchain pages.

Deploy depends on #362 (test4) due to mismatched `avl` versions on test3.

## Contributors Checklist

- [x] Added new tests, or not needed, or not feasible
- [x] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [x] Updated the official documentation or not needed
- [x] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [x] Added references to related issues and PRs
- [x] Provided any useful hints for running manual tests

## Maintainers Checklist

- [x] Checked that the author followed the guidelines in `CONTRIBUTING.md`
- [x] Checked the conventional-commit (especially PR title and verb, presence of `BREAKING CHANGE:` in the body)
- [x] Ensured that this PR is not a significant change or confirmed that the review/consideration process was appropriate for the change

### CODE REVIEW

File: examples/gno.land/r/gnoland/pages/admin.gno [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+)
+
+var (
+	adminAddr     std.Address
+	moderatorList avl.Tree
+	inPause       bool
+)
+
+func init() {
+	// adminAddr = std.GetOrigCaller() // FIXME: find a way to use this from the main's genesis.
```
Comment (ajnavarro) [inline examples/gno.land/r/gnoland/pages/admin.gno:17]: shall we open an issue for this?

File: examples/gno.land/r/gnoland/pages/admin.gno [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+func AdminAddModerator(addr std.Address) {
+	assertIsAdmin()
+	moderatorList.Set(addr.String(), true)
+}
+
+func AdminRemoveModerator(addr std.Address) {
+	assertIsAdmin()
+	moderatorList.Set(addr.String(), false) // FIXME: delete instead?
```
Comment (ajnavarro) [inline examples/gno.land/r/gnoland/pages/admin.gno:38]: Are these FIXMEs something to be done before merging?

File: examples/gno.land/r/gnoland/pages/admin.gno (hunk 1 above)
Comment (moul) [reply examples/gno.land/r/gnoland/pages/admin.gno:17]: Just check, I think we already have one, it’s related to the thing we said about having a way to skip the need of faucet for local development, where we could imagine having a detection of accounts on the FS to automatically craft a genesis.

Note that we already have a system which patches the genesis to have a dynamic hostname.

File: examples/gno.land/r/gnoland/pages/admin.gno (hunk 2 above)
Comment (moul) [reply examples/gno.land/r/gnoland/pages/admin.gno:38]:
```suggestion
	moderatorList.Set(addr.String(), false) // XXX: delete instead?
```

Yeah, not a fixme, more like an open question about best practices, thank you.

**Review by waymobetta — APPROVED** [review]

**Review by ajnavarro — APPROVED** [review]

File: examples/gno.land/r/gnoland/pages/admin.gno (hunk 1 above)
Comment (zivkovicmilos) [reply examples/gno.land/r/gnoland/pages/admin.gno:17]: Can't we set this value in the `init` method (constructor)?

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯

File: examples/gno.land/r/gnoland/pages/admin.gno (hunk 1 above)
Comment (moul) [reply examples/gno.land/r/gnoland/pages/admin.gno:17]: We can, but not with « auto-genesis »; we need to have the auto-genesis taking an account as argument to simulate the initial uploaded.

**Pull request merged**